### PR TITLE
New package: py-tf-slim

### PR DESCRIPTION
**Pull request by qwertos**

### CODE REVIEW

**Comment by sethrj** [comment]
@qwertos Would you be willing to add yourself as maintainer to this new package?

File: var/spack/repos/builtin/packages/py-tf-slim/package.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    version('1.1.0', sha256='964cde4b7728a408dcd5c841ab6b93d95137ab4b60db28b10400f86286bfeb8b')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-absl-py@0.2.2:', type='build')
+    depends_on('py-six', type='run')
+    depends_on('py-wrapt', type='run')
+    depends_on('py-gast', type='run')
+    depends_on('py-termcolor', type='run')
+    depends_on('flatbuffers', type='run')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-tf-slim/package.py:26]: Where are you finding these deps? I see six being imported but not the others.

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-tf-slim/package.py:26]: With out them, `import tf_slim` will throw a `module not found` error at runtime.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-tf-slim/package.py:26]: Can you share the stack trace?

Comment (alalazo) [reply var/spack/repos/builtin/packages/py-tf-slim/package.py:26]: @qwertos

Comment (qwertos) [reply var/spack/repos/builtin/packages/py-tf-slim/package.py:26]: I'm getting some non-deterministic issues in trying to replicate this again for the trace. I'm going to mark this as draft for now.

**Comment by adamjstewart** [comment]
Is this PR still a WIP or should we close?

**Pull request closed without merging**